blocks/columns/columns.css

```diff
@@ -100,6 +100,18 @@ main .columns-container h4 {
   margin-bottom: 0;
 }
 
+.columns.content-band.icon-list.add-bottom-space > div {
+  margin-bottom: var(--spacer-layout-04);
+}
+
+.columns.content-band.icon-list.add-bottom-space > div > div > p {
+  justify-content: left;
+}
```

Comment:
This is not needed as all picture/img within each row of the block should be aligned left.

.columns.content-band > div > div > p.only-picture {
  justify-content: left;
}

```suggestion
```

Reply:
@proeung - have made these changes.